assets/constants.json

```diff
@@ -7,5 +7,6 @@
     "Mathematics", "Medicine", "Music", "Philosophy", "Physics", "Poetry",
     "Probability", "Programming", "Puzzles", "Reading", "Spanish", "Sport",
     "Statistics", "Trigonometry", "Welcome"],
-  "DISABLED_EXPLORATION_IDS": ["5"]
+  "DISABLED_EXPLORATION_IDS": ["5"],
+  "NEW_STATE_TEMPLATE":{"content":[{"type":"text","value":""}],"interaction":{"answer_groups":[],"confirmed_unclassified_answers":[],"customization_args":{"rows":{"value":1},"placeholder":{"value":"Type your answer here."}},"default_outcome":{"dest":"(untitled state)","feedback":[],"param_changes":[]},"fallbacks":[],"id":"TextInput"},"param_changes":[]}
```

Comment:
NEW_STATE_TEMPLATE originates from [editor.py](https://github.com/oppia/oppia/blob/develop/core/controllers/editor.py#L55), using [exp_domain.State.NULL_INTERACTION_DICT](https://github.com/oppia/oppia/blob/develop/core/controllers/editor.py#L61), so we can have both the constants NEW_STATE_TEMPLATE and NULL_INTERACTION_DICT in constants.json that will work for front-end as controller logic will generate the _nested_ constants in proper format. but how will we generate the _nested_ constants for tests as they are directly fetched from the static constants.json file.